gamestonk_terminal/main_helper.py

```diff
@@ -601,7 +637,91 @@ def view(other_args: List[str], s_ticker: str, s_interval, df_stock):
         return
 
 
+def check_api_keys():
+    """Check api keys and if they are supplied"""
+
+    AVAILABLE_KEYS = {
```

Comment:
I think it would be nice to also do the "testing" part. Most of these APIs allow you to check if the API key is correct or not, so that would be a nice to have.

Reply:
So I was envisioning a --test flag that could do the basic checks but I am really not sure on the best checks for all of these without either wasting api calls or supplying the wrong one.

Reply:
I think most APIs have something like that to check key without requiring to use a call

Reply:
Ill go through and see which ones I can do and then document which ones need to be filled